airflow/models/xcom.py

```diff
@@ -509,40 +513,40 @@ def get_many(
             message = "Passing 'execution_date' to 'XCom.get_many()' is deprecated. Use 'run_id' instead."
             warnings.warn(message, RemovedInAirflow3Warning, stacklevel=3)
 
-        query = session.query(BaseXCom).join(BaseXCom.dag_run)
+        query = select(BaseXCom).join(BaseXCom.dag_run)
```

Comment:
one small suggestion, if it possible could you rename variable `query` to `stmt` it would make it a bit easier to found is we migrate this or not when look at the code